internal/util/vault.go

```diff
@@ -41,7 +41,7 @@ const (
 	vaultDefaultPassphrasePath = ""
 
 	// vault request headers
-	vaultTokenHeader     = "X-Vault-Token" // nolint: gosec, #nosec
+	vaultTokenHeader     = "X-Vault-Token" // #nosec:G101, credential not vulnerable.
```

Comment:
these are not hard-coded credentials, just the name of the variable that holds the token (credential)